rpcs3/Loader/TRP.cpp

```diff
@@ -141,3 +151,56 @@ void TRPLoader::RenameEntry(const char *oldname, const char *newname)
 		}
 	}
 }
+
+bool TRPLoader::TrimEntries()
+{
+	// Rename or discard certain entries based on the files found
+	const size_t kTargetBufferLength = 31;
+	char target[kTargetBufferLength + 1];
+	target[kTargetBufferLength] = 0;
+	strcpy_trunc(target, fmt::format("TROP_%02d.SFM", static_cast<s32>(g_cfg.sys.language)));
+
+	if (ContainsEntry(target))
+	{
+		RemoveEntry("TROPCONF.SFM");
+		RemoveEntry("TROP.SFM");
+		RenameEntry(target, "TROPCONF.SFM");
+	}
+	else if (ContainsEntry("TROP.SFM"))
+	{
+		RemoveEntry("TROPCONF.SFM");
+		RenameEntry("TROP.SFM", "TROPCONF.SFM");
+	}
+	else if (!ContainsEntry("TROPCONF.SFM"))
+		return false;
+
+	// Discard unnecessary TROP_XX.SFM files
+	for (s32 i = 0; i <= 18; i++)
+	{
+		strcpy_trunc(target, fmt::format("TROP_%02d.SFM", i));
+		if (i != g_cfg.sys.language)
+		{
+			RemoveEntry(target);
+		}
+	}
+	return true;
+}
+
+u64 TRPLoader::GetFileSize()
+{
+	if (!m_header.trp_file_size)
```

Comment:
Personally prefer this style, but every other if statement is wrapped in the code.